src/test-utils/mocks/mockGqlResponse.ts

```diff
@@ -0,0 +1,59 @@
+const mockGqlResponse = {
```

Comment:
This is fine for now, but you'll probably want to set up a factory later on, to have different responses per test